### ISSUE TITLE
Navigation component does not work without javascript

### ISSUE THREAD

**Issue by minusf** [a11y-collective/demos#44]
I have a couple of questions about the navigation component.

1. Unfortunately it seems that while the navigation component (https://github.com/a11y-collective/demos/blob/master/en/advanced-accessible-components/components/navigation/navigation.html) might be completely a11y friendly, there doesn't seem to be any graceful degradation in place when javascript is not available, making all sub-menus inaccessible for everyone.

2. I see many proponents of a11y menus advocate for using _only_ click to open sub-menus (essentially all top level menu items being buttons) to stop the confusion if a menu item opens a sub-menu vs acting as a menu item itself. There are some techniques to enable click to open menus without javascript, so even though the aria attributes are not set correctly, at least the menus work.

3. Hover not being available on mobile devices, having a hover mode makes the desktop and mobile versions inconsistent.
